setup.cfg

```diff
@@ -32,7 +32,7 @@ install_requires =
     particle>=0.16.0
     scipy>=1.0.0
     torch>=1.0.0
-    uproot3>=3.14.1
+    uproot4>=4
```

Comment:
```suggestion
    uproot>=4.0.0
```

Given that `uproot4` is never going to have `uproot5` in it, maybe it is better to just only use the name `uproot`.

Reply:
Indeed, this should not be `uproot4`.